internal/cmd/manager/instance/initdb/cmd.go

```diff
@@ -40,10 +53,20 @@ func NewCmd() *cobra.Command {
 	var postInitSQLStr string
 	var postInitApplicationSQLStr string
 	var postInitTemplateSQLStr string
+	var postInitApplicationSQLSecretRefs string
+	var postInitApplicationSQLConfigMapRefs string
 
 	cmd := &cobra.Command{
 		Use: "init [options]",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			_, err := client.New(ctrl.GetConfigOrDie(), client.Options{
+				Scheme: scheme,
+			})
+			if err != nil {
+				log.Error(err, "Error while getting ")
+				return err
+			}
+
```

Comment:
Given that these secrets and configmaps are needed only in the init job we could think of mounting them directly in the pod, maybe in a single directory with an index (e.g. "00-....sql", 01-... etc) so that we can keep track of the ordering. This way we would avoid requiring permissions to access the configmaps and the secrets, a client and we could pass a flag specifying the folder only, instead of the whole list of secrets with keys. What do you think, @Anhui-tqhuang ?

Reply:
yeah, that's a good point